spot_driver/spot_driver/spot_ros2.py

```diff
@@ -1104,7 +1106,7 @@ def main(args=None):
         spot_ros.spot_wrapper = SpotWrapper(spot_ros.username, spot_ros.password, spot_ros.ip, spot_ros.name,
                                             spot_ros.wrapper_logger, spot_ros.start_estop.value, spot_ros.estop_timeout.value,
                                             spot_ros.rates, spot_ros.callbacks, spot_ros.use_take_lease,
-                                            spot_ros.get_lease_on_action, spot_ros.continually_try_stand)
+                                            spot_ros.get_lease_on_action, spot_ros.continually_try_stand, spot_ros.rgb_cameras.value)
```

Comment:
There should be a corresponding submodule change for `SpotWrapper` with this I think?

Reply:
sorry for not mentioning here..

https://github.com/bdaiinstitute/spot_wrapper/pull/15

Reply:
Right but you also need to change the submodule pointer in this repository to point to the updated version in `spot_wrapper`.